src/templates/fundraising/index.js

```diff
@@ -133,26 +152,20 @@ export default {
     Initialize a transparent and accountable crowdfunding campaign for your
     organization.
   `,
-  userGuideUrl: 'https://help.aragon.org/',
+  userGuideUrl: 'https://fundraising.aragon.black/',
   sourceCodeUrl: 'https://github.com/AragonBlack/fundraising',
   registry: 'aragonpm.eth',
   apps: [
+    {
+      appName: 'aragon-fundraising.aragonpm.eth',
+      label: 'Aragon Fundraising',
+    },
+    { appName: 'agent.aragonpm.eth', label: 'Agent [Reserve Pool]' },
     { appName: 'token-manager.aragonpm.eth', label: 'Board Token' },
     { appName: 'voting.aragonpm.eth', label: 'Board Voting [Multisig]' },
     { appName: 'token-manager.aragonpm.eth', label: 'Share Token' },
-    { appName: 'voting.aragonpm.eth', label: 'Share Voting [Democracy]' },
+    { appName: 'voting.aragonpm.eth', label: 'Share Voting' },
     { appName: 'finance.aragonpm.eth', label: 'Finance' },
-    { appName: 'agent.aragonpm.eth', label: 'Agent [Reserve Pool]' },
-    { appName: 'presale.aragonpm.eth', label: 'Presale' },
-    {
-      appName: 'batched-bancor-market-maker.aragonpm.eth',
-      label: 'Market Maker [Batched Bancor]',
-    },
-    { appName: 'tap.aragonpm.eth', label: 'Tap' },
-    {
-      appName: 'aragon-fundraising.aragonpm.eth',
-      label: 'Aragon Fundraising [API]',
-    },
```

Comment:
It's not really necessary to put all the other apps in this list; just fundraising should be enough.